{{cookiecutter.repository_name}}/operations/evaluate.py

```diff
@@ -57,13 +57,13 @@ def evaluate(config):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--eval_batch_size", type=int, default=64)
-    parser.add_argument("--n_workers", default=2, type=int)
+    parser.add_argument("--n_workers", type=int, default=2)
+    parser.add_argument("--cuda", type=bool, choices=[True, False], default=True)
     args = parser.parse_args()
 
     config = vars(args)
     config.update(
-        seed=1,
-        use_cuda=torch.cuda.is_available(),
+        seed=2464,
```

Comment:
Also, shouldn't train and evaluate set the same seeds? Imo evaluate can load the seed from the train config?

Reply:
Why? Seeds don't work that way